### PR TITLE
fix: availability table migration not running in MySQL strict mode

### PR DESCRIPTION
**Pull request by AxonC**
- Adds nullable to the 'from' & 'to' columns on the availability table. This fixes the issue by allowing null timestamps (whereas the default MySQL stance is not to do so)﻿

### CODE REVIEW

File: database/migrations/2020_02_03_204857_create_availability_table.php [hunk 1]
```diff
@@ -16,8 +16,10 @@ public function up()
         Schema::create('availability', function (Blueprint $table) {
             $table->bigIncrements('id');
             $table->unsignedInteger('user_id');
-            $table->timestamp('from');
-            $table->timestamp('to');
+            $table->timestamp('from')->nullable()
+                ->comment('Nullable to enforce compatibility with MYSQL strict mode.');
+            $table->timestamp('to')->nullable()
+                ->nullable('Nullable to enforce compatibility with MYSQL strict mode.');
```
Comment (CalumTowers) [inline database/migrations/2020_02_03_204857_create_availability_table.php]: This should be a comment

**Review by CalumTowers — CHANGES_REQUESTED** [review]
If someone has availability, why would either from or to be null?

File: database/migrations/2020_02_03_204857_create_availability_table.php (hunk 1 above)
Comment (AxonC) [reply database/migrations/2020_02_03_204857_create_availability_table.php]: > If someone has availability, why would either from or to be null?

MySQL doesn't like letting in a zero value for timestamp by default, hence why this migration was failing with MySQL strict mode enabled.

Comment (AxonC) [reply database/migrations/2020_02_03_204857_create_availability_table.php]: > This should be a comment

Fixed

**Review by atoff — APPROVED** [review]

**Pull request merged**